services/translate.cfc

```diff
@@ -0,0 +1,58 @@
+component {
+
+  variables.service = 'translate';
+
+  public any function init(
+      required any api,
+      required struct settings
+  ) {
+      variables.api = arguments.api;
+      return this;
+  }
+
+  public any function translate(required string text, string sourceLang='es', string targetLang='en' ){
+    var payload = {};
+    payload[ 'SourceLanguageCode' ] = sourceLang;
+    payload[ 'TargetLanguageCode'] = targetLang;
+    payload[ 'Text' ] = text;
+
+    var apiResponse = apiCall( payload = serializeJSON(payload) );
+    if ( apiResponse.statusCode == 200 ) {
+      apiResponse[ 'data' ] = deserializeJSON(apiResponse.rawData).TranslatedText;
+    }
+    return apiResponse;
+  }
+
+  public string function getHost(
+      required string region
+  ) {
+      return variables.service & '.' & region & '.amazonaws.com';
+  }
+
+  private any function apiCall(
```

Comment:
It looks to me like the translate API is structured similarly to the dynamodb API - there is only one action here, but, like the dynomodb API, everything is implemented using POST to the root API path. So I would prefer to see this method structured similarly to the `apiCall()` method in the `dynamodb.cfc`:  https://github.com/jcberquist/aws-cfml/blob/master/services/dynamodb.cfc#L415-L431 - I think the calls to `serializeJSON()` and `deserializeJSON()` could also be moved into `apiCall()`.